src/l1-resources/stream-connection/README.md

```diff
@@ -0,0 +1,69 @@
+# stream-connection
+
+The official [MongoDB Atlas](https://www.mongodb.com/) AWS CDK resource for Node.js.
+
+> AWS CDK [L1 construct] and data structures for the [AWS CloudFormation Registry] type `MongoDB::Atlas::StreamConnection`.
+[L1 construct]: https://docs.aws.amazon.com/cdk/latest/guide/constructs.html
+[AWS CloudFormation Registry]: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/registry.html
+
+> **NOTE**: 
```

Comment:
https://github.com/mongodb/awscdk-resources-mongodbatlas/pull/233/files#r1517995266 would apply here as well.

Reply:
Put it in the Usage section